tutorials/SingleQubitGates/SingleQubitGates.ipynb

```diff
@@ -342,7 +342,7 @@
    "source": [
     "### <span style=\"color:blue\">Demo: Pauli Gates</span>\n",
     "\n",
-    "The following cell contains code demonstrating how to apply gates in Q#, using the Pauli $X$ gate as an example. It sets up a series of quantum states, and then shows the result of applying the $X$ gate to each one. To run the demo, run the next cell using `Ctrl+Enter` (`⌘+Enter` on a Mac), then run the cell after it to see the output.\n",
+    "The following cell contains code demonstrating how to apply gates in Q#, using the Pauli $X$ gate as an example. It sets up a series of quantum states, and then shows the result of applying the $X$ gate to each one. To run the demo, run the next cell using `Ctrl+Enter` (`⌘+Enter` on a Mac).\n",
```

Comment:
I would keep this phrasing - we're adding the note to clarify the next cell but we still need to run it. This is just the 2nd tutorial in which the learner sees Q# and how it works in the notebook, so it makes sense to reiterate that you only get output after the 2nd cell.

Reply:
I have added a new cell after the code, but before the `%trace` call to reiterate how to execute the code. But if you want I can also bring this back